drawee/src/main/java/com/facebook/drawee/drawable/RoundedBitmapDrawable.java

```diff
@@ -322,7 +345,10 @@ private void updatePaint() {
       mIsShaderTransformDirty = true;
     }
     if (mIsShaderTransformDirty) {
-      mPaint.getShader().setLocalMatrix(mTransform);
+      // Draw the bitmap at the correct scale
+      if (!mTransform.isIdentity()) {
+        mPaint.getShader().setLocalMatrix(mTransform);
```

Comment:
Do you have to reset the matrix otherwise? I.e. does this work if you switch from a complex to the circular case?